watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Networking/BaseRequest.java

```diff
@@ -30,9 +30,9 @@
     private Response.Listener listener;
     private Response.Listener errorListener;
 
-//    private static final String baseURL = "http://155.41.97.153:3000/api/v1/";
+    private static final String baseURL = "http://192.168.0.104:3000/api/v1/";
 //    private static final String baseURL = "https://intense-reaches-1457.herokuapp.com/api/v1/";
-    private static final String baseURL = "https://floating-bayou-8262.herokuapp.com/api/v1/";
+//    private static final String baseURL = "https://floating-bayou-8262.herokuapp.com/api/v1/";
```

Comment:
Can you delete the local address line (as in, not just comment it out)? Also should delete the intense-reaches url.